woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -3035,15 +3035,21 @@ def test_validate_logical_types(sample_df):
             "email_2": {4: pd.NA, 5: "bad_email"},
         }
     )
-
     expected = expected.astype(
         {
-            "url": "string",
-            "email": "string",
-            "email_2": "string",
+            "url": "string[pyarrow]",
+            "email": "string[pyarrow]",
+            "email_2": "string[pyarrow]",
         }
     )
-
+    if _is_spark_dataframe(df):
```

Comment:
Because `pyspark` doesn't support this yet right?

Reply:
is there any concern that `validate_logical_types()` returns a different type?

Reply:
@Cmancuso @bchen1116 I'm a little but confused/concerned with this. Was going to try to take a closer look to see if I could understand why this is the case, but haven't had a chance yet.

Reply:
I can also try to look into it when I get back next week! I tried yesterday/today, but didn't end up having time to

Reply:
Looked into this a little bit - the dtype change is happening during the concat operation here (coming from pyspark): https://github.com/alteryx/woodwork/blob/6332317c111993be4ad7f12686a077dae4341edf/woodwork/table_accessor.py#L1148

The series dtypes are all `string[pyarrow]` up until that point for pandas, Dask and Spark dataframes. For some reason the spark concat operation changes the dtype, while the pandas and Dask concat operations don't (maybe a spark bug/issue?)

However, looking at this test a bit more closely I don't think we care about the dtypes of `actual`. In this case `actual` is just a DataFrame that contains the invalid values. This DataFrame does not have Woodwork initialized and as such we shouldn't really have any expectation for getting certain dtypes back in that df as long as the values are correct.

I'm thinking we could update this test completely to remove the type casting on expected dataframe and compare `expected` to `actual` with `pd.testing.assert_frame_equal` using `check_dtype=False` since the dtype isn't so critical in this case.

@jeff-hernandez Do you have any strong opinions on this?